tests/snapshot_iterator.go

```diff
@@ -0,0 +1,42 @@
+package main
```

Comment:
I think you have to change package name (main -> test)

Reply:
Sorry some late.
This code has been changed to verify-tools.